### ISSUE TITLE
[Low] [Internal] Tasks - Error occurs when creating subtask with external user on task assigned to said user

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#36925]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.43.3
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):** u.onyeukwu94+0219@gmail.com
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

## Action Performed
1. Create group chat with 2 other accounts
2. Assign task in group to external user not included group chat
3. Log into the external user's account and open task details of assigned task
4. Press “+” in compose and create task

## Expected Result:
User can create subtask in task details of assigned chat

## Actual Result:

- Error occurs when substack is created and substack remains grey. 
- Error message is :`Unexpected error create task. Please try again later.`
- After clearing error, returning to LHN and returning to assigned task, Substack is still created

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [x] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/d088264b-2876-40bd-b439-0f05b5f3ca3f

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01fdc5f4ea9fa3552b</li>
        <li>Upwork Job ID: 1761064624098820096</li>
        <li>Last Price Increase: 2024-03-15</li>
    </ul>
</details>

**Comment by lanitochka17** [comment]
We think that this bug might be related to #vip-vsp
CC @quinthar

**Comment by lanitochka17** [comment]
@bz FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by laurenreidexpensify** [comment]
waiting for proposals - looks like automation failed on adding the `external` label

**Comment by brunovjk** [comment]
I believe it is a issue with the backend,
We are getting a `genericCreateTaskFailureMessage: 'Unexpected error create task, please try again later.'` response to `CreateTask` call, and we shouldn't, since _"User can create subtask in task details of assigned chat"_. Can anyone confirm please? Thanks.

**Comment by laurenreidexpensify** [comment]
Waiting for proposals

**Comment by Pujan92** [comment]
@laurenreidexpensify It seems like a backend issue as it(`CreateTask`) is responding with the error. In the group chat report, I see those created tasks. Can we pull an internal engineer to confirm?

<img width="1135" alt="Screenshot 2024-02-27 at 18 12 57" src="https://github.com/Expensify/App/assets/14358475/c816dd7e-4816-4be0-a9f8-30fdc48cb165">

https://github.com/Expensify/App/assets/14358475/33565fea-dfcc-488d-a54c-f91cfccd22a3

**Comment by laurenreidexpensify** [comment]
Asking internally https://expensify.slack.com/archives/C066HJM2CAZ/p1709530744995619

**Comment by laurenreidexpensify** [comment]
bumped slack thread

**Comment by laurenreidexpensify** [comment]
Still seeing if proposals come in

**Comment by brunovjk** [comment]
> Asking internally https://expensify.slack.com/archives/C066HJM2CAZ/p1709530744995619

Hi @laurenreidexpensify :D sorry to ask, but I don't have access. Could we confirm if this is **not** a [backend issue](https://github.com/Expensify/App/issues/36925#issuecomment-1966474974)? Thanks.

**Comment by laurenreidexpensify** [comment]
Confirmed this is internal, remains a Low Priority for now

**Comment by laurenreidexpensify** [comment]
Waiting for internal eng to have capacity to pick up

**Comment by laurenreidexpensify** [comment]
Waiting for internal eng to have capacity to pick up

**Comment by Pujan92** [comment]
Waiting for internal eng

**Comment by laurenreidexpensify** [comment]
Internal eng

**Comment by laurenreidexpensify** [comment]
Still waiting on internal eng

**Comment by laurenreidexpensify** [comment]
Waiting on internal eng

**Comment by laurenreidexpensify** [comment]
Waiting on internal eng

**Comment by laurenreidexpensify** [comment]
Waiting on internal eng

**Comment by laurenreidexpensify** [comment]
Waiting on internal eng

**Comment by Pujan92** [comment]
Unassigning myself as I believe it requires a backend change and doesn't need C+.

**Comment by laurenreidexpensify** [comment]
Waiting on internal eng

**Comment by laurenreidexpensify** [comment]
Waiting on internal eng

**Comment by laurenreidexpensify** [comment]
Waiting on internal eng

**Comment by laurenreidexpensify** [comment]
@garrettmknight you're my parental leave handover buddy on this one - thanks! 

TLDR is that we're waiting for an internal engineer to pick up. I'll leave it to you to discern if you think it's worth keeping open at all or considering closing, but we've had no one look at this in ~3 months :(

**Comment by garrettmknight** [comment]
Let's close - this is minor and not tied to anything on the roadmap for the moment.
